app/web/src/components/ProjectProfile/ProjectProfile.tsx

```diff
@@ -35,12 +35,12 @@ const KeyValue = ({
   if (!children || hide) return null
   return (
     <div>
-      <div className="text-ch-blue-600 font-fira-code flex text-sm">
+      <div className="text-ch-blue-600 font-fira-code flex text-sm whitespace-nowrap">
```

Comment:
"created on" was wraping to a new line even though it had lots of space, not sure if related or another safari quirk.

Reply:
turn off whitespace to see this diff correctly

Reply:
Yeah I saw that happening in your initial screenshot, I bet it had to do with the width of these elements compressing as flex children or something. We can keep an eye on it when getting the mobile view to look good, `nowrap` can cause layout weirdness on narrow devices.